core/src/impl/Kokkos_Profiling.cpp

```diff
@@ -807,21 +809,20 @@ VariableValue make_variable_value(size_t id, double val) {
   variable_value.value.double_value = val;
   return variable_value;
 }
-Experimental::VariableValue make_variable_value(size_t id, const char* val) {
+Experimental::VariableValue make_variable_value(size_t id,
+                                                const std::string& val) {
   VariableValue variable_value;
-  variable_value.type_id            = id;
-  variable_value.value.string_value = val;
+  variable_value.type_id = id;
+  strncpy(variable_value.value.string_value, val.c_str(), 63);
```

Comment:
Do not use magic numbers

Reply:
Yup, fixed that one via bd5eb14, good comment. There are magic numbers elsewhere in Profiling, I'll fix those in a separate PR though